lib/rules/color-hex-case/index.js

```diff
@@ -21,7 +21,8 @@ const meta = {
 	deprecated: true,
 };
 
-const HEX = /^#[0-9A-Za-z]+/;
+const HEX = /^#[\da-z]+/i;
+const CONTAINS_HEX = /#[\da-z]+/i;
```

Comment:
[add] Does it make sense to reduce characters like this?

```suggestion
const HEX = /^#[\da-f]+/i;
const CONTAINS_HEX = /#[\da-f]+/i;
```

Reply:
These rules specifically check for `a-z`.
I was surprised by this, but didn't want to change the logic.

Reply:
I see. Let's keep it as-is. 👍🏼